packages/lib-classifier/src/store/SubjectStore.spec.js

```diff
@@ -39,22 +39,34 @@ describe('Model > SubjectStore', function () {
   describe('Actions > advance', function () {
     describe('with a full queue', function () {
       let rootStore
-      let previousSubject
       let initialSize
 
       before(function () {
         rootStore = setupStores()
       })
 
       beforeEach(function () {
-        previousSubject = rootStore.subjects.resources.values().next().value.id
         initialSize = rootStore.subjects.resources.size
         rootStore.subjects.advance()
       })
 
       it('should reduce the queue size by one', function () {
         expect(rootStore.subjects.resources.size).to.equal(initialSize - 1)
       })
+    })
+
+    describe('setting an active subject', function () {
```

Comment:
This is a bit of a hack, since it duplicates the previous describe block. Can you see why the tests fail if we have more than one expectation inside a describe block? 

For instance, this block of tests will fail if we add an expectation that advancing the queue should reduce the queue size by one. 

I also found that with two expectations inside a describe block, the second expectation always fails even if I swapped the order of the two.

Reply:
The description here doesn’t really describe the test, which is to call advance on a full queue of subjects (same as the previous describe block.) Setting the active subject is an expectation of what should happen.